### ISSUE TITLE
is it possible to publish this lib over jitpack?

### ISSUE THREAD

**Issue by WuglyakBolgoink** [su-vikas/conbeerlib#2]

**Comment by blundell** [comment]
Yes please: https://medium.com/@danielkao/how-to-publish-an-android-library-to-jcenter-a31231b47241

**Comment by su-vikas** [comment]
@WuglyakBolgoink  sorry for late reply, I need to check and if its not much work, I will do it. 
@blundell Thanks for the link. I will go through it.

**Comment by blundell** [comment]
any luck releasing?

**Comment by blundell** [comment]
![](https://media0.giphy.com/media/3o6MboXbEtIjCSsU6I/200.gif)

**Comment by su-vikas** [comment]
@blundell I am extremely sorry for not able to release this on Jitpack. i did try to spend sometime to understand and put it on jitpack, but didn't reach to conclusion. I will try it again soon and hopefully release it. I am not much experienced with such stuff, so there is an initial inertia to get moving.
